topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/tutorial.bib

```diff
@@ -0,0 +1,83 @@
+#  1) Find the DOI for the article you want to cite
+#  2) Go to https://doi2bib.org and fill in the DOI
+#  3) Copy the resulting bibtex entry into this file
+#
+# To cite the example below, in your tutorial.md file
+# use {% cite Batut2018 %}
+#
+# If you want to cite an online resourse (website etc)
+# you can use the 'online' format (see below)
+#
+# You can remove the examples below
+
+@article{Trapnell2014,
+  doi = {10.1038/nbt.2859},
+  url = {https://doi.org/10.1038/nbt.2859},
+  year = {2014},
+  month = mar,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {32},
+  number = {4},
+  pages = {381--386},
+  author = {Cole Trapnell and Davide Cacchiarelli and Jonna Grimsby and Prapti Pokharel and Shuqiang Li and Michael Morse and Niall J Lennon and Kenneth J Livak and Tarjei S Mikkelsen and John L Rinn},
+  title = {The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells},
+  journal = {Nature Biotechnology}
+}
+
+@article{Bacon2018,
+  doi = {10.3389/fimmu.2018.02523},
+  url = {https://doi.org/10.3389/fimmu.2018.02523},
+  year = {2018},
+  month = nov,
+  publisher = {Frontiers Media {SA}},
+  volume = {9},
+  author = {Wendi A. Bacon and Russell S. Hamilton and Ziyi Yu and Jens Kieckbusch and Delia Hawkes and Ada M. Krzak and Chris Abell and Francesco Colucci and D. Stephen Charnock-Jones},
+  title = {Single-Cell Analysis Identifies Thymic Maturation Delay in Growth-Restricted Neonatal Mice},
+  journal = {Frontiers in Immunology}
+}
+
+@online{monocle3,
+  author = {cole-trapnell-lab},
+  title = {monocle3},
+  url = {https://cole-trapnell-lab.github.io/monocle3/},
+  urldate = {2022-09-23}
+}
+
+@online{Alevin tutorial,
+  author = {Wendi Bacon, Jonathan Manning},
+  title = {Generating a single cell matrix using Alevin},
+  url = {% link topics/transcriptomics/tutorials/scrna-case_alevin/tutorial.md %},
+  urldate = {2022-09-23}
+}
+
+@online{Combining datasets tutorial,
+  author = {Wendi Bacon, Jonathan Manning},
+  title = {Combining datasets after pre-processing},
+  url = {% link topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md %},
+  urldate = {2022-09-23}
+}
+
+@online{Filter&plot tutorial,
+  author = {Wendi Bacon},
+  title = {Filter, Plot and Explore Single-cell RNA-seq Data},
+  url = {% link topics/transcriptomics/tutorials/scrna-case_basic-pipeline/tutorial.md %},
+  urldate = {2022-09-23}
+}
+
+@online{Trajectory analysis Python,
+  author = {Wendi Bacon},
+  title = {Trajectory Analysis using Python (Jupyter Notebook) in Galaxy},
+  url = {% link topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md %},
+  urldate = {2022-09-23}
+}
+
+@online{Trajectory analysis,
+  author = {Julia Jakiela},
+  title = {Trajectory analysis slide deck},
+  url = {% link topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/slides.html %},
+  urldate = {2022-09-23}
```

Comment:
I think these can be removed (it looks like links are being used everywhere) and those will be easier for users than citations

Reply:
done!